ext/opcache/jit/zend_jit_helpers.c

```diff
@@ -727,6 +727,7 @@ static zval* ZEND_FASTCALL zend_jit_fetch_dim_rw_helper(zend_array *ht, zval *di
 static zval* ZEND_FASTCALL zend_jit_fetch_dim_w_helper(zend_array *ht, zval *dim)
 {
 	zend_long hval;
+	zend_ulong idx;
```

Comment:
Just changing hval to zend_ulong should be fine here.

Reply:
Hmm, OK.